docs/custom-flows/sign-out.mdx

```diff
@@ -68,8 +71,11 @@ Please note that the sign-out flow only deactivates the current session. Other v
         await clerk.load()
 
         if (clerk.user) {
+          // Attach signOut function to the sign-out button
```

Comment:
Be wary of unnecessary comments - a developer should know what this line is doing
```suggestion
```